Analytics/add_startup_admin.py

```diff
@@ -27,7 +27,7 @@ class AddStartupAdmin(Command):
     def __init__(self):
         """
         Create a local sqlalchemy engine that connects to the database
-        defined in settings.py
+        defined in settings.py.bak
```

Comment:
Since settings.py files are no longer used, do we remove or update these comments?

Reply:
I think those aren't needed. I think it was suggested elsewhere.